### PR TITLE
mail2kvanitha/c03-docker01

### PR DESCRIPTION
**Pull request by mail2kvanitha**
# Description

Exercise (e.g. C01-E01): mail2kvanitha/c03-docker01

Student Name: Vanitha Kaliyaperumal

> A brief explanation of what has been done.
Create a docker image and run a container from this docker image created.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker01/mail2kvanitha/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  > docker run -it da-terraform:c03-docker-e01
+
+  Terraform v0.12.20
+
+ Your version of Terraform is out of date! The latest version
+ is 0.12.28. You can update by downloading from https://www.terraform.io/downloads.html
+
+```
+
```
Comment (denstorti) [inline classes/03class/exercises/c03-docker01/mail2kvanitha/README.md:50]: Can you output the size of the final image? 

Should be less than 80MB

File: classes/03class/exercises/c03-docker01/mail2kvanitha/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<!-- Don't change anything below this point-->
+<!-- Before commiting, remove both commented lines--> 
+***
+Answer for exercise [c03-docker01](https://github.com/devopsacademyau/academy/blob/af3225a3436f263164e8daebc6bbd1ef3122b900/classes/03class/exercises/c03-docker01/README.md
```
Comment (denstorti) [inline classes/03class/exercises/c03-docker01/mail2kvanitha/README.md:54]:
```suggestion
Answer for exercise [c03-docker01](https://github.com/devopsacademyau/academy/blob/af3225a3436f263164e8daebc6bbd1ef3122b900/classes/03class/exercises/c03-docker01/README.md)
```

File: classes/03class/exercises/c03-docker01/mail2kvanitha/README.md (hunk 1 above)
Comment (mail2kvanitha) [reply classes/03class/exercises/c03-docker01/mail2kvanitha/README.md:50]: Please see below the image size.

> docker image ls da-terraform:c03-docker-e01
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
da-terraform        c03-docker-e01      c50fc0cc7915        2 days ago          56MB

File: classes/03class/exercises/c03-docker01/mail2kvanitha/README.md (hunk 2 above)
Comment (kikobr82) [reply classes/03class/exercises/c03-docker01/mail2kvanitha/README.md:59]: can you fix this missing `)`?

Comment (mail2kvanitha) [reply classes/03class/exercises/c03-docker01/mail2kvanitha/README.md]: Fixed the error. Please review now.

File: classes/03class/exercises/c03-docker01/mail2kvanitha/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker01/mail2kvanitha/Dockerfile:1]: can you change the tag to something that is not latest?

**Pull request merged**